src/uint/sqrt.rs

```diff
@@ -9,7 +9,7 @@ impl<const LIMBS: usize> Uint<LIMBS> {
     /// Uses Brent & Zimmermann, Modern Computer Arithmetic, v0.5.9, Algorithm 1.13
     ///
     /// Callers can check if `self` is a square by squaring the result
-    pub const fn sqrt(&self) -> Self {
+    pub const fn sqrt_vartime(&self) -> Self {
```

Comment:
Ok, I'll do that instead. This means it will still be renamed in 0.6.0, correct? Do you have any automated procedure to handle these deprecation markers and make sure you don't forget any on release?

Reply:
Fixed.

Reply:
Yes, it will be removed in the next release.

There's no automated check for that. It's just something you have to do.